src/third-party/PhishingController.ts

```diff
@@ -23,6 +23,43 @@ export interface EthPhishingResponse {
   whitelist: string[];
 }
 
+/**
+ * @type EthPhishingDetectConfig
+ *
+ * Interface defining expected input to PhishingDetector.
+ * @property allowlist - List of approved origins (legacy naming "whitelist")
+ * @property blocklist - List of unapproved origins (legacy naming "blacklist")
+ * @property fuzzylist - List of fuzzy-matched unapproved origins
+ * @property tolerance - Fuzzy match tolerance level
+ */
+export interface EthPhishingDetectConfig {
+  allowlist: string[];
+  blocklist: string[];
+  fuzzylist: string[];
+  tolerance: number;
+  name: string;
+  version: number;
+}
+
+/**
+ * @type EthPhishingDetectResult
+ *
+ * Interface that describes the result of the `test` method.
+ * @property name - Name of the config on which a match was found.
+ * @property version - Version of the config on which a match was found.
+ * @property result - Whether a domain was detected as a phishing domain. True means an unsafe domain.
+ * @property match - The matching fuzzylist origin when a fuzzylist match is found. Returned as undefined for non-fuzzy true results.
+ * @property type - The field of the config on which a match was found.
+ */
+export interface EthPhishingDetectResult {
+  name: string;
+  version: string;
```

Comment:
Whoops, an earlier suggestion of mine made these mandatory, but they actually aren't present for negative results

```suggestion
  name?: string;
  version?: string;
```